frontend/src/test/scala/bloop/engine/FileWatchingSpec.scala

```diff
@@ -0,0 +1,148 @@
+package bloop.engine
+
+import java.io.{ByteArrayOutputStream, PrintStream}
+import java.nio.file.Files
+
+import bloop.Project
+import bloop.cli.{CliOptions, Commands}
+import bloop.tasks.{CompilationHelpers, ProjectHelpers}
+import bloop.tasks.ProjectHelpers.{RootProject, noPreviousResult, withState}
+import org.junit.Test
+
+class FileWatchingSpec {
+  @scala.annotation.tailrec
+  final def readCompilingLines(target: Int, msg: String, out: ByteArrayOutputStream): Int = {
+    Thread.sleep(100) // Wait 10ms for the OS's file system
```

Comment:
100ms?

Reply:
Yes, to avoid reading all the time until it appears (since it happens concurrently).

Reply:
Sorry I keep being unclear. The code says 100ms, the comment says 10.

Reply:
I'll make an effort 💃

Reply:
Hahaha, makes sense, removing it :smile: